k8s/src/main/scala/io/buoyant/k8s/README.md

```diff
@@ -0,0 +1,2 @@
+This package contains representations of objects returned by multiple versions of the Kubernetes API.
+Version-specific objects should go in sub-packages (see v1.scala).
```

Comment:
ahh, yeah good catch.  should be in `k8s/` or a comment on `k8s/src/main/scala/io/buoyant/k8s/package.scala`